docs: add issues templates (#116)
ImpeccableHQ/swapr-info: CONTRIBUTING.md

# Contribution Guidelines

A Contribution Guideline shall facilitate the foundation for a collaborative ground for every contributor working on Swapr codebase(s).

## Project Management Software

We use [ZenHub](https://zenhub.com), an agile project management which is tightly integrated with GitHub. Contact one of the core contributors to invite you to the ZenHub board.

## Tickets

- Every task needs have an estimated ticket
- If you see something during work which is out of scope of the ticket:
  1. make a new ticket or reopen a ticket if exists.
  2. finish the current ticket first.
  3. If not depending on the other ticket, make a new branch from the **develop** branch, not the branch you are working on.
  4. No ticket is needed, but a branch, if you can do it under one hour. If you see that it takes longer, make a ticket with estimate.
  5. You can restimate your tickets if you see its much more work. But do not use it to track hours. It's NOT time tracking.

## Git Branches

- One ticket, one branch.
- If the ticket involes subtasks
  - Create a parent ticket
  - Branch out from parent ticket and merge from subtickets
  - Merge parent ticket to `develop`

> - Use /feature/, /bug/, or /chore/ (chore to fix typo and little stuff)

Reviewer: ```suggestion
- Use `feature/{description}`, `bug/{description}`, or `chore/{description}` (chore to fix typo and little stuff)
```

- Avoid working on the `main` branch unless absolutely necessary. Branch names should be named after what they do.

> - sub-branch like "Feature/stufspecial/otherstuff" should not happen. You can work for yourself in this structur, but please don't get others to work in your sub-branch (It's a sign that something is off. We add to much complexity to non complex stuff. The interface software is still a simple interface)

Reviewer: ```suggestion
- sub-branch like `feature/stufspecial/otherstuff` should not happen. You can work for yourself in this structure, but please don't get others to work in your sub-branch (It's a sign that something is off. We add too much complexity to non-complex stuff.)
```

Some more toughts on branches see [Phil Hord's answer on Stack Overflow](https://stackoverflow.com/a/6065944/2151050).

## Pull Requests (PR)

- PRs should target `develop` branch,
- A subtask PR should target parent branch.
- Draft PRs should be used when a PR is Work In Progress (WIP).

> - If you make a PR from `feature/stufspecial/otherstuff` to `feature/stufspecial/` you should pull it yourself.

Reviewer: ```suggestion
- If you make a PR from `feature/stufspecial/otherstuff` to `feature/stufspecial` you should pull it yourself.
```
